packages/synapse-react-client/src/lib/style/components/_download-cart-page.scss

```diff
@@ -145,10 +145,6 @@
   .checkbox label {
     font-weight: normal;
   }
-  .page-progress-percent {
-    align-self: start;
-    margin-bottom: 15px;
-  }
```

Comment:
Removed this styling -- I couldn't find a place where PageProgress was used in the DownloadCartPage